controllers/matchControllers.js

```diff
@@ -1,5 +1,9 @@
 const Match = require('../models/Match');
+const User = require('../models/User');
+const customError = require('../helpers/customErrorHandler');
+const mongoose = require('mongoose');
 
+// just for checking reason
 exports.getMatches = async (req, res, next) => {
```

Comment:
I think this it two calls in one, one is get matches and one is get match singular. also i dont think two res.jsons will work in one call